recipe/recipe.yaml

```diff
@@ -0,0 +1,113 @@
+context:
+  version: "3.2.9"
+
+package:
+  name: scs
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/bodono/scs-python/archive/refs/tags/${{ version }}.tar.gz
+    sha256: 56973e33629a40de41648591cc288096803d09c86c7a0ef8291f33a157f8c312
+  # include submodule (not in github tarball due to dear-github/dear-github#214)
+  - git: https://github.com/cvxgrp/scs.git
+    rev: c8297172633bcb3a10d4781a19d4769ce5282d29
+    target_directory: scs_source
+  - git: https://github.com/python/pythoncapi-compat.git
+    rev: fde4d3457d7c1e6f7d09afeae5afd2218fbb2cae
+    target_directory: scs/pythoncapi-compat
+
+build:
+  number: 2
+  string: ${{ ("mkl_py" if blas_impl == "mkl" else "default_py") }}${{ python | version_to_buildstring }}h${{ hash }}_${{ build_number }}
+  script:
+    - if: unix
+      then:
+        - if: blas_impl == "mkl"
+          then:
+            - MESON_ARGS="-Dlink_mkl=true ${MESON_ARGS}"
+        - python -m pip install . -vv -Csetup-args=${MESON_ARGS// / -Csetup-args=}
+      else:
+        - set "EXTRA="
+        - if: win and blas_impl == "mkl"
+          then:
+            - set "EXTRA=-Csetup-args=-Dlink_mkl=true"
+        - python -m pip install . -vv %EXTRA%
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - libblas * *${{ blas_impl }}
+        - liblapack * *${{ blas_impl }}
```

Comment:
This is not as previously requested. If I already go to the trouble of providing a concrete diff as a suggestion, please don't make me repeat myself.